### ISSUE TITLE
Feature Request: Render index.html if available when rendering a directory path

### ISSUE THREAD

**Issue by tswast** [RangerMauve/datmobile#1]
I love the progress you've made on this so far. I'm filing this issue because I didn't see it in the README checklist yet.

When I browse to or dat://www.timswast.com/ or dat://www.timswast.com/blog/, I'd like it to actually render dat://www.timswast.com/index.html or dat://www.timswast.com/blog/index.html. On the current Play Store version, I see a directory listing instead.

![image](https://user-images.githubusercontent.com/247555/53975218-33945f80-40b9-11e9-8cb9-2022168eb5ac.png)

**Comment by RangerMauve** [comment]
Yes! I'm working on getting the content loading through a webview and I'll be adding this feature as part of that.

I was playing around with getting that stuff working [here](https://github.com/RangerMauve/load-dat-page/blob/master/contentLoader.js#L231) on the web.

**Comment by tswast** [comment]
Great! That resolution order looks good to me. You might want to add

`Try loading the folder + index.md`

after

`Try loading the folder + index.html`

in the "it's a folder case".

**Comment by RangerMauve** [comment]
I based the order from what I saw in Beaker's code. There should probably be a DEP for the resolution somewhere for future implementations, though. 😂
